app/Utils/LatexHelper.php

```
@@ -0,0 +1,71 @@
+    /**
+     * Converts a single character to a \symbol{...} command
+     * @param  string  $char
+     * @return string
+     */
+    private static function convertCharToSymbol(string $char): string
+    {
+        if ($char == ' ') {
+            return ' ';
+        }
+        return "\\symbol{" . mb_ord($char) . "}";
+    }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'convertCharToSymbol'. (undefined)

(UnusedPrivateMethod)